tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -455,28 +456,47 @@ public function makeDavRequest(
 	 * @param string|null $height
 	 *
 	 * @return void
+	 * @throws GuzzleException
+	 * @throws JsonException
 	 */
 	public function downloadPreviews(string $user, ?string $path, ?string $doDavRequestAsUser, ?string $width, ?string $height):void {
-		$user = $this->getActualUsername($user);
-		$doDavRequestAsUser = $this->getActualUsername($doDavRequestAsUser);
 		$urlParameter = [
 			'x' => $width,
 			'y' => $height,
 			'forceIcon' => '0',
 			'preview' => '1'
 		];
-		$this->response = $this->makeDavRequest(
+		$davVersion = $this->getDavPathVersion();
+		if ($davVersion === WebDavHelper::DAV_VERSION_SPACES) {
+			$fileId = urlencode(strstr($this->getFileIdForPath($doDavRequestAsUser ?? $user, $path), '!', true));
+
+			$eTag = str_replace('"', '', $this->getFileEtagForPath($doDavRequestAsUser ?? $user, $path));
```

Comment:
Let's not use etag, it will only be extra work for the tests. The server does nothing with `c` parameter as of now.

CC @ScharfViktor
```suggestion
```

Reply:
Yes, @amrita-shrestha sorry for the confusion. I double-checked with the backend developers, the backend doesn't use the "c" parameter, so we don't need to include it in apiTest

Reply:
Do I have to wait until the backend developer removes 'c' parameter or I can move on with this PR @ScharfViktor

Reply:
No, you don't have to. The backend hasn't made use of `c` (etag) yet so no need to remove anything in the backend. You can continue on this PR.